StripePaymentSheet/StripePaymentSheet.xcodeproj/project.pbxproj

```diff
@@ -338,6 +338,9 @@
 		A0D3076C2EA9484D008940E5 /* NSAttributedStringStripeSnapshotTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = A0D3076B2EA9484D008940E5 /* NSAttributedStringStripeSnapshotTests.swift */; };
 		A1AC7034E778F81B8758A653 /* OHHTTPStubsSwift in Frameworks */ = {isa = PBXBuildFile; productRef = 20076FBD56C42E259EF62F2B /* OHHTTPStubsSwift */; };
 		A30E2EC92E9E958B00E982DB /* STPClientAttributionMetadata+PaymentSheet.swift in Sources */ = {isa = PBXBuildFile; fileRef = A30E2EC82E9E958B00E982DB /* STPClientAttributionMetadata+PaymentSheet.swift */; };
+		A30E31432EABEFBE00E982DB /* AttestationConfirmationChallenge.swift in Sources */ = {isa = PBXBuildFile; fileRef = A30E31422EABEFBE00E982DB /* AttestationConfirmationChallenge.swift */; };
```

Comment:
You can remove these changes, the xcodeproj will now pull in source files automatically